deployment/huggingface/yolos-seq2seq.ipynb

```diff
@@ -0,0 +1,1699 @@
+        "resources = Resources(cpu=1., memory='4Gi')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "XG597pl9fgZW"
+      },
+      "source": [
+        "## 8. Model Endpoint"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "-bqBsgLNizcs"
+      },
+      "outputs": [],
+      "source": [
+        "transformers_cache_env = {'TRANSFORMERS_CACHE': '/tmp'}"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "gjbfpxDYhWqX"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint = client.get_or_create_endpoint('yolos-hugging-face')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "4psmdLh4i3LR",
+        "outputId": "cf2c326a-4610-4b19-9a67-49e61c7418bb"
+      },
+      "outputs": [],
+      "source": [
+        "%%time\n",
+        "endpoint.update(\n",
+        "    model,\n",
+        "    autoscaling = autoscaling,\n",
+        "    resources = resources,\n",
+        "    env_vars = transformers_cache_env,\n",
+        "    wait = True\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "3KQEQzt5hXFR"
+      },
+      "outputs": [],
+      "source": [
+        "deployed_model = endpoint.get_deployed_model()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ePYbeR9ffgVi"
+      },
+      "source": [
+        "## 9. Predictions"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "uwFAheCHffW2"
+      },
+      "outputs": [],
+      "source": [
+        "url = 'http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/'"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "tG_VRyK9jEeP"
+      },
+      "outputs": [],
+      "source": [
+        "req = requests.get(url)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Zobr2rpzjElo"
+      },
+      "outputs": [],
+      "source": [
+        "soup = BeautifulSoup(req.text, 'lxml')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4Uy-i4CujOr2"
+      },
+      "outputs": [],
+      "source": [
+        "urls = []\n",
+        "\n",
+        "for link in soup.find_all('a'):\n",
+        "    href = link.get('href')\n",
+        "  \n",
+        "    if href.endswith('.jpg'):\n",
+        "        url = f\"http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/{href}\"\n",
```

Comment:
See my comment in another PR about trying to reuse the above `url` here

https://github.com/VertaAI/examples/pull/50#discussion_r1060891886